src/types/Todos.tsx

```diff
@@ -0,0 +1,5 @@
+export type Todos = {
```

Comment:
Use interface prefer.
Type has unique ability but if you don`t need them use interface. It is sampled in scaled and has more interesting features.